python/hail/dataset.py

```diff
@@ -3845,8 +3845,9 @@ def pca(self, scores, loadings=None, eigenvalues=None, k=10, as_array=False):
     @typecheck_method(k=integral,
                       maf=numeric,
                       block_size=integral,
-                      min_kinship=numeric)
-    def pc_relate(self, k, maf, block_size=512, min_kinship=-float("inf")):
+                      min_kinship=numeric,
+                      desire=enumeration("phi", "phik2", "phik2k0", "all"))
+    def pc_relate(self, k, maf, block_size=512, min_kinship=-float("inf"), desire="all"):
```

Comment:
as Konrad mentioned, probably there should be a docstring for the "desire" parameter?